CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -317,6 +317,21 @@ public function upgrade_4_7_18($rev) {
     $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_4_7_19($rev) {
+    $query = "SELECT id FROM civicrm_financial_account WHERE opening_balance <> 0 OR current_period_opening_balance <> 0";
```

Comment:
By default this Columns were set to 0. Therefore i checked if this columns have values other than 0 then don't drop column.